src/movies.css

```diff
@@ -0,0 +1,12 @@
+.movies-container {
+    padding: 10px;
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-gap: 20px;
+    border: 1px solid #000;
+}
+
+.grid-item > img {
```

Comment:
We can potentially refactor this file into SASS for readability, but this is a good start to getting the styling we need.